plugins/generator-1.21.1/neoforge-1.21.1/plant.definition.yaml

```diff
@@ -75,6 +75,10 @@ templates:
     writer: json
     condition: "plantType %= normal"
     name: "@MODASSETSROOT/blockstates/@registryname.json"
+  - template: json/block_states.json.ftl

```

Comment:
Since this template is defined 3 times, you can optimize this with "condition_any" to accept any of the 3 conditions